.changeset/remove-pre-flight.md

```
@@ -0,0 +1,20 @@
+<!-- Auto-generated by gen-changeset. Do not edit manually. -->
+---
+"agent-gauntlet": minor
+---
+
+### Features
```

markdownlint-cli2 (0.20.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)